src/App.tsx

```diff
@@ -1,51 +1,84 @@
 import React from 'react';
 import './App.scss';
+import classnames from 'classnames';
+import goodsFromServer from './goods';
 
-// import goodsFromServer from './goods';
+type State = {
+  selectedGood: string,
+};
 
-export const App: React.FC = () => {
-  return (
-    <main className="App">
-      <header className="App__header">
-        <h1 className="App__title">
-          Jam is selected
-        </h1>
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    selectedGood: '',
+  };
 
-        <button
-          type="button"
-          className="App__clear"
-        >
-          Clear
-        </button>
-      </header>
+  render(): React.ReactNode {
+    const { selectedGood } = this.state;
 
-      <ul>
-        <li className="Good">Dumplings</li>
-        <li className="Good">Carrot</li>
-        <li className="Good">Eggs</li>
-        <li className="Good">Ice cream</li>
-        <li className="Good">Apple</li>
-        <li className="Good">Bread</li>
-        <li className="Good">Fish</li>
-        <li className="Good">Honey</li>
-        <li className="Good Good--active">Jam</li>
-        <li className="Good">Garlic</li>
-      </ul>
+    const selgoods = selectedGood ? `${selectedGood} ${' '} is selected` : 'No goods selected';
```

Comment:
```suggestion
    const heading = selectedGood
      ? `${selectedGood} ${' '} is selected`
      : 'No goods selected';
```
heading name better suits here
and better to format code like this - it is easier to read